modules/standard/AdvancedIters.chpl

```diff
@@ -50,8 +63,27 @@ iter dynamic(c:range(?), chunkSize:int, numTasks:int=0) {
 
 // Parallel iterator
 
-// Leader iterator
+/*
+
+  :arg c: The range to iterate over. The length of the range must be greater than zero.
+  :type c: range(?)
+
+  :arg chunkSize: The size of chunks to be yielded to each thread. Must be greater than zero.
+  :type chunkSize: int
 
+  :arg numTasks: The number of tasks to use. Must be >= zero. If this argument has the value 0, it will use the value indicated by ``dataParTasksPerLocale``.
```

Comment:
These long lines can be wrapped like so:
```
  :arg numTasks: The number of tasks to use. Must be >= zero. If this argument
                 has the value 0, it will use the value indicated by
                 ``dataParTasksPerLocale``.
```
and it will render the same.